case-lib/lib.sh

```diff
@@ -685,8 +685,8 @@ ipc4_used()
 
     # If /sys/module/snd_sof_pci/parameters/ipc_type exists
     # If the value of file ipc_type is:
-    # 0: DUT runs IPC3 mode, ipc_used return 1(false)
-    # 1: DUT runs IPC4 mode, ipc4_used return 0(true)
+    # 0: DUT runs IPC3 mode, is_ipc4 return 1(false)
+    # 1: DUT runs IPC4 mode, is_ipc4 return 0(true)
```

Comment:
The comment does not make any sense
 0: DUT runs IPC3 mode, **is_ipc4 return 1(false)**
1: DUT runs IPC4 mode, **is_ipc4 return 0(true)**
if ipc_type is 1, this function returns 1 (is_ipc4 is true)

Reply:
I just checked, for IPC3 mode, the ipc_type is `-1`(updated). 
I don't see any problems here. if `ipc_type=1`, `is_ipc4` function returns 0. which means `is_ipc4` is true.
We use `|| `here.
```shell
 [ "$ipc_type" -eq 1 ] || {
        return 1
    }
    return 0
```

Reply:
OK, I see. Thanks